scripts/spectra/spectra.py

```diff
@@ -63,19 +68,18 @@ def read_fomo_outfile(filename):
 
 
 def lorentzian(
-    x,
-    x0,
-    delta,
+    x, x0, delta,
 ):
 
     return 0.5 * delta / np.pi * 1.0 / ((x - x0) ** 2 + (0.5 * delta) ** 2)
 
 
+def gaussian(x, x0, delta):
+    return np.exp(-np.power(x - x0, 2.0) / (2 * np.power(delta, 2.0)))
+
+
```

Comment:
Add documentation. Write a one-line docstring describing what the function does, annotate the function signature with the types you are passing it, and describe teh passed parameters. The function documentation should look like the rest of the well-documented functions in the package. If you need help, ask @hayleyweir , she knows how to do this now :)

Reply:
I'm ok to do this, but I thought we were only documenting the actual aimsprop package in this way? This is just a script that was already there.